webapp/README.md

```diff
@@ -22,6 +22,16 @@ npm run clean --workspaces --if-present
 
 To install dependencies for a workspace, simply run `npm install` from this folder as you would do normally. Most packages' dependencies will be included in the root `node_modules`, and all packages' dependencies will appear in the `package-lock.json`. A `node_modules` will only be created inside a package if one of its dependencies conflicts with that of another package.
 
+NOTE: if you were on Linux ARM64 silicon (eg. virtual Linux on Mac M1) and encountered below errors when installing `optipng-bin` package:
```

Comment:
Mind removing this section? While this is good information to keep around, I think most of our docs refer to using the `makefile` instead of using `npm install` directly which would do this automatically. The only reason this section talks about `npm install` was because I was trying to make sure people don't try running `npm install` inside of a workspace

Reply:
Pleasure, removed this.